File: Results/DER/Translation/translation_output/w_test/gpt-4/codenet/python/java/p03407.java
Certainly! Below is the translated Java code that reads three integers from the standard input and prints "Yes" if the sum of the first two integers is greater than or equal to the third integer, otherwise it prints "No".

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int A = scanner.nextInt();
        int B = scanner.nextInt();
        int C = scanner.nextInt();

        if (A + B < C) {
            System.out.println("No");
        } else {
            System.out.println("Yes");
        }

        scanner.close();
    }
}
```
You can use this Java code to test with the input `50 100 120` and it should provide the expected output `Yes`.

File: Results/DER/Translation/translation_output/w_test/gpt-4/codenet/java/python/p03407.py
Here's the equivalent Python code to the provided Java code:

```python
def two_coins_solve(a, b, c):
    return "Yes" if a + b >= c else "No"

if __name__ == "__main__":
    inputs = input().split()
    a, b, c = map(int, inputs)
    result = two_coins_solve(a, b, c)
    print(result)
```

Now, suppose you input `50 100 120` as in the test case, the expected output should be "Yes".
